main/src/library/Fixpoint/Ast/Datalog.flix

```diff
@@ -22,22 +22,23 @@ mod Fixpoint.Ast.Datalog {
     use Fixpoint.SubstitutePredSym
     use Fixpoint.SubstitutePredSym.substitute
     use Fixpoint.Ast.Ram.RamSym
-    use Fixpoint.Ast.Shared.Denotation
-    use Fixpoint.Ast.Shared.PredSym
+    use Fixpoint.Ast.Shared.{Denotation, PredSym}
+    use Fixpoint.Ast.Shared.{BoxedDenotation => Denotation}
```

Comment:
Historically Magnus has been against use-renaming, but otherwise it looks good

Reply:
Yeah. But I suppose this code will be deleted anyway soonish, so.